packages/studio/index.d.ts

```diff
@@ -181,6 +181,14 @@ declare module "@foxglove/studio" {
     /** Actions the panel may perform related to the user's current layout. */
     readonly layout: LayoutActions;
 
+    /**
+     * Identifies the semantics of the data being played back, such as which topics or parameters
+     * are semantically meaningful or normalization conventions to use. This typically maps to a
+     * shorthand identifier for a robotics framework such as "ros1", "ros2", or "ulog". See the MCAP
+     * profiles concept at <https://github.com/foxglove/mcap/blob/main/docs/specification/appendix.md#well-known-profiles>.
+     */
+    readonly profile: string;
```

Comment:
I think we should have this field be undefined (rather than an empty string) if no profile is available.

```suggestion
    readonly profile?: string;
```